DiagManager/CustomDiagnostics/SQL Base/Multicopy.ps1

```diff
@@ -0,0 +1,22 @@
+﻿param(
```

Comment:
You really don't need this file. You should remove it from project. The one in Pristine folder covers all Custom Collectors